### ISSUE TITLE
[BUG]: xdebug crash on breakpoint after using phalcon

### ISSUE THREAD

**Issue by BenKennish** [phalcon/cphalcon#14882]
**Describe the bug**
im having problems getting phalcon and xdebug to play nicely.  xdebug works on my WAMP stack when im doing simple, non-phalcon stuff, but if i set a breakpoint after the code has done some phalcon stuff, xdebug/php seems to crash

**To Reproduce**

```php
$fruits = ["Apple", "Banana", "Cherry"];
foreach ($fruits as $fruit)
{
    // setting a breakpoint on the following line works fine
    echo $fruit;
}
$obj = new \Phalcon\Config(['testing'=>'ben']);
// xdebug/PHP seems to crash if i set a breakpoint on the following line and when the debugger reaches it
echo 'Done';
```
the crash results in a 500 Internal Server Error for the web browser and the following messages in the various log files...

Apache error log:
```
[Wed Feb 26 11:15:31.171658 2020] [fcgid:warn] [pid 18276:tid 6888] (OS 109)The pipe has been ended.  : [client 127.0.0.1:62149] mod_fcgid: get overlap result error
[Wed Feb 26 11:15:31.171658 2020] [core:error] [pid 18276:tid 6888] [client 127.0.0.1:62149] End of script output before headers: xdebug.php
```

Last few lines of xdebug.log:
```
[7864] <- stack_get -i 37
[7864] -> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="https://xdebug.org/dbgp/xdebug" command="stack_get" transaction_id="37"><stack where="{main}" level="0" type="file" filename="file:///C:/htdocs/sandbox/xdebug.php" lineno="10"></stack></response>

[7864] <- context_get -i 38 -d 0 -c 0
```

**Expected behavior**
PHP should not crash

**Details**
 - Phalcon version: 3.2.2
 - PHP Version: 7.1.11-Win32-VC14-x64
 - Operating System: Windows 10
 - Installation type: Zip file
 - Server: Apache

**Comment by Jeckerson** [comment]
What version of xdebug?

**Comment by BenKennish** [comment]
> What version of xdebug?
2.9.2-7.1-vc14

**Comment by BenKennish** [comment]
> What version of xdebug?
2.9.2-7.1-vc14

**Comment by Jeckerson** [comment]
Seems that fix in `2.8.0` version was fixed only for `PHP >=7.2`

> Fixed bug #1699: Crash during debugging Phalcon project

* https://xdebug.org/updates#x_2_8_0
* https://bugs.xdebug.org/bug_view_page.php?bug_id=00001699

**Comment by Jeckerson** [comment]
Can you try to update PHP version to >=7.2 and check if problem still there?

**Comment by BenKennish** [comment]
> Fixed bug #1699: Crash during debugging Phalcon project

Is this the correct bug number?  When I click it, I get taken to a pull request entitled "Fix handling of AUTOINCREMENT in addColumn()/modifyColumn()" and cannot see anything relevant

> Can you try to update PHP version to >=7.2 and check if problem still there?

I will give it a go when I get some time.

**Comment by Jeckerson** [comment]
> Is this the correct bug number?
No, it is Github automatically create link, use second link from my comment.

**Comment by Jeckerson** [comment]
@BenKennish any news?

**Comment by BenKennish** [comment]
@Jeckerson I thought I'd fixed it by updating to PHP 7.3.14 but then I started getting weird issues where PhpStorm would suggest that xdebug was breaking at points where I hadn't set breakpoints (e.g. I'd set a breakpoint on line 123 and it would actually break on line 140.)  I've been too busy to look into it since.  I'm guessing you can probably close this issue.

**Comment by Jeckerson** [comment]
OK, feel free to reopen if problem persists in recent version of PHP.
